src/Microsoft.VisualStudio.Editor.UnitTests/ApplicationDesigner/ApplicationDesignerPanelTests.cs

```diff
@@ -0,0 +1,545 @@
+using System;
```

Comment:
This goes for all the source files - Do you need to copyright notice before they go into GitHub?